types/n3/index.d.ts

```diff
@@ -1,15 +1,15 @@
-// Type definitions for N3 1.4
```

Comment:
Version bump should not be needed, DT will automatically perform a patch update.